### PR TITLE
Translating kubectl.md

### PR DESCRIPTION
**Pull request by hurf**
This is the main page of kubectl commands

Partially #20

### CODE REVIEW

**Comment by hurf** [comment]
This pr needs carefully review, for the followings may depend on this one.

**Comment by hurf** [comment]
@resouer @linfan @iT2afL0rd

File: docs/user-guide/kubectl/kubectl.md [hunk 1]
```diff
@@ -23,75 +19,61 @@ kubectl
...
-  -s, --server="": The address and port of the Kubernetes API server
-      --stderrthreshold=2: logs at or above this threshold go to stderr
-      --token="": Bearer token for authentication to the API server.
-      --user="": The name of the kubeconfig user to use
-      --username="": Username for basic authentication to the API server.
-      --v=0: log level for V logs
-      --vmodule=: comma-separated list of pattern=N settings for file-filtered logging
+      --alsologtostderr[=false]: 同时输出日志到标准错误控制台和文件。
+      --api-version="": 和服务端交互使用的API版本。
+      --certificate-authority="": 用以进行证书权威性认证的.cert文件路径。
```
Comment (resouer) [inline docs/user-guide/kubectl/kubectl.md:28]: 权威性认证 ---> 授权认证

Comment (hurf) [inline docs/user-guide/kubectl/kubectl.md:28]: 我解释下，这个证书不是用来认证的，下面的那个client-certificate才是。如果client-certificate是权威机构颁发的，则不需要certificate-authority，如果是自签名的话就需要这个，否则会报certificate issued by unknown authority的错。所以我把这里翻译成证书权威性认证。这样说的通不？

Comment (resouer) [inline docs/user-guide/kubectl/kubectl.md:28]: sgtm

**Comment by resouer** [comment]
generally, lgtm

**Pull request merged**